packages/lmi/src/lmi/llms.py

```diff
@@ -833,6 +882,18 @@ async def acompletion(self, messages: list[Message], **kwargs) -> list[LLMResult
         used_model = completions.model or self.name
         results: list[LLMResult] = []
 
+        # Extract cached token fields and calculate cost
+        cache_read, cache_creation = parse_cached_usage(
+            completions.usage  # type: ignore[attr-defined]
```

Comment:
```suggestion
            getattr(completions, "usage", None)
```

Just resolving the type ignore